Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -55,9 +55,17 @@ void HTMLLinkElement::inserted()
 
 void HTMLLinkElement::parse_attribute(FlyString const& name, String const& value)
 {
+    // https://html.spec.whatwg.org/multipage/links.html#rel-icon
     if (name == HTML::AttributeNames::rel) {
+        // 4.6.7 Link types
+        //
+        // To determine which link types apply to a link, a, area, or form element,
+        // the element's rel attribute must be split on ASCII whitespace.
+        // The resulting tokens are the keywords for the link types that apply to that element.
+        //
+        // Keywords are always ASCII case-insensitive, and must be compared as such.
         m_relationship = 0;
-        auto parts = value.split_view(' ');
+        auto parts = value.to_lowercase().split_view(' ');
```

Comment:
Is this safe? We create a new string with `to_lowercase` but `split_view` uses stringviews so the string might be gone at that point?

Reply:
The lowercased result should then be stored in a variable, which then lasts until the end of the current scope, I guess?